src/modules/85-cv/pages/monitored-service/components/Configurations/components/Service/Service.utils.ts

```diff
@@ -60,7 +60,7 @@ export const onSave = async ({
   formik,
   onSuccess
 }: {
-  formik: FormikContext<any>
+  formik: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
  formik: FormikContextType<unknown>
```